docs/source/faq.rst

```diff
@@ -135,7 +135,9 @@ input ordered as latitude/longitude  (typically with the EPSG dataset), however,
 internally PROJ expects an longitude/latitude ordering for all projections. This
 is generally hidden for users but in a few cases it is exposed at the surface
 level of PROJ, most prominently in the :program:`proj` utility which expects
-longitude/latitude ordering of input date (unless :option:`proj -r` is used).
+longitude/latitude ordering of input data (unless :option:`proj -r` is used).
+(And `geod`, which on the contrary expects latitude/longetude, and lacks any
```

Comment:
\`geod\` should probably be :program:\`geod\`.

Also, putting an entire sentence in parenthesis looks strange, especially as it bumps oddly against the parenthesis at the end of the previous sentence. I suggest something like
```rst
... (unless :option:`proj -r` is used), while :program:`geod`,
on the contrary, expects latitude/longitude, and does not support the `-r` option.
```

Also note the correction longetude -> longitude